core/src/main/java/nl/nn/adapterframework/filesystem/FileSystemListener.java

```diff
@@ -284,6 +284,10 @@ public Message extractMessage(F rawMessage, Map<String,Object> threadContext) th
 			if (getMessageType().equalsIgnoreCase("contents")) {
 				return getFileSystem().readFile(rawMessage, getCharset());
 			}
+			if (getMessageType().equalsIgnoreCase("info")) {
```

Comment:
Nu messageType nog geen Enum is (en misschien ook niet wordt), moet deze waarde gedocumenteerd worden in de ibisdoc van setMessageType()